### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by LaskoA**
Please check my solution, thank you :

)

### CODE REVIEW

File: taxi/views.py [hunk 1]
```diff
@@ -6,7 +6,8 @@
 from django.contrib.auth.mixins import LoginRequiredMixin
 
 from .models import Driver, Car, Manufacturer
-from .forms import DriverCreationForm, DriverLicenseUpdateForm, CarForm
+from .forms import DriverCreationForm, DriverLicenseUpdateForm, CarForm, DriverSearchForm, CarSearchForm, \
+    ManufacturerSearchForm
```
Comment (Abnormaltype) [inline taxi/views.py:10]: fix indentation

File: taxi/views.py [hunk 2]
```diff
@@ -32,10 +33,25 @@ def index(request):
...
+        context["search_form"] = ManufacturerSearchForm(initial={"name": name})
+
+        return context
+
+    def get_queryset(self):
+        name = self.request.GET.get("name")
+
+        if name:
+            return self.queryset.filter(name__icontains=name)
+        return self.queryset
```
Comment (Abnormaltype) [inline taxi/views.py:53]: duplicate of code) try to fix it

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

**Comment by LaskoA** [comment]
Fixes were done, please check again. Thank you)

**Review by Abnormaltype — APPROVED** [review]
Well done!

**Pull request closed without merging**